### ISSUE TITLE
Suggestion - Towing/Impound prices

### ISSUE THREAD

**Issue by MAGiCxPOTATOE** [InZidiuZ/legacy-assets-issue-tracker#752]
### Name

xxfaze_magicx_potatoexx360

### Summary

I've been a mechanic in the server for quite some time. With that I would do quite a lot of tow calls or even just drive around towing the odd time if I am bored. I was thinking that maybe the prices for Impounding a vehicle as a tow driver and as a person retrieving their car from it could be changed. 
Point 1: The cars dannies are given when they first fly in gives the same amount as a local car(I have confirmed it is not a local car).
Point 2: Other various vehicles from around the city should give a bit more than a local car e.g. trailers, trash trucks.
Point 3: Even the price you get from towing a persons car could be lowered as well as the price for getting a car out of the impound.
Point 4: Since there are less restarts recently, there is a much bigger build up of random vehicles that have been abandoned.

### Reason

Point 1 Reason: 90% of the cars I tow are these danny cars, so like asea, emperor , rust truck etc. Maybe there are less active mechanics in general but there seems to be no one that i have seen that goes around towing without being called, and even still im pretty sure most tow calls just get ignored unless they are from cops. 
Changing the price of local cars e.g. to a consistent 100$ would encourage tow drivers to do tow full time, sort of like how Burger shot has behind counter and deliveries. And tow is one of the jobs that is good for both the server and the people in the server as it clears a decent amount of clutter.
Point 2 Reason: The only one I can really think of that is relevant is garbage trucks, but some other things could be heli's, trucker trailers, delivery trucks. These are some of the biggest vehicles a.k.a. nuisances and would be good to have a reason for these to be impounded.
Point 3 Reason: Most people that are established enough in the city know not to leave their cars out, and even if it gets impounded. The drive or treck to the impound is bad enough and some new people to the city may not know about the tow/impound system, not sure if it is in the brochure or not. Then 500$ to get back can destroy some dannies bank.(this could be the reason that the danny cars impound for so little, maybe they retrieve for less too?)
Point 4 Reason: More towing will lead to less cars for people to crash into, less clutter and may even help performance a teeny tiny bit.

My sanity gave up halfway through writing this so it may not make much sense, apologies.
TL;DR: City is economy driven, Tow could do with having a somewhat more consistent money output e.g. 100$ for every vehicle, local or not, then decrease un-impount fee.

**Comment by InZidiuZ** [comment]
This is the wrong issue tracker. This one is for assets only. Use [this one](https://github.com/InZidiuZ/op-framework-issue-tracker/issues) instead.
